.github/workflows/build-test-linux.yml

```diff
@@ -6,7 +6,7 @@ on:
   push:
     branches: [master, main]
     paths-ignore:
-      - '**.md'
+      - '**/*.md'
```

Comment:
While you are at it, I would consider adding a `not-sh, not-go files, not Makefile and definitely not-terraform files`; however, its not a blocker from me.

Reply:
We can add them, but also I would consider the likelihood of those being changed without something else being changed that would require running unit tests. I think Markdown files very typically can be updated independently of other things. I don't see many meaningful changes to a bash script or makefile or terraform file that wouldn't directly be associated with code that we need to build and unit test.